package/MDAnalysis/coordinates/TRJ.py

```diff
@@ -115,10 +115,10 @@
 .. Links
 
 .. _AMBER: http://ambermd.org
-.. _AMBER TRJ format: http://ambermd.org/formats.html#trajectory
+.. _AMBER TRJ format: http://www.webcitation.org/query?url=http%3A%2F%2Fambermd.org%2Fformats.html&date=2018-02-11 
 ..    The formats page was archived as
 ..    http://www.webcitation.org/query?url=http%3A%2F%2Fambermd.org%2Fformats.html&date=2018-02-11
-..    Use the archived version if the original disappears. [orbeckst]
+..    Use the archived version if the original (http://ambermd.org/formats.html#trajectory) disappears. [orbeckst]
```

Comment:
Given that the archived page does not contain the TRJ format, let's delete the whole comment.
```suggestion
```